storage-node/src/services/webApi/app.ts

```diff
@@ -82,7 +79,8 @@ export async function createApp(config: AppConfig): Promise<Express> {
       validateRequests: true,
       operationHandlers: {
         basePath: path.join(__dirname, './controllers'),
-        resolver: OpenApiValidator.resolvers.modulePathResolver,
+        resolver: (basePath: string, route: RouteMetadata, apiDoc: OpenAPIV3.Document) =>
+          asyncHandler(OpenApiValidator.resolvers.modulePathResolver(basePath, route, apiDoc)),
```

Comment:
It catches the errors thrown by the async route handlers and passes it to the default [error handler](https://github.com/zeeshanakram3/joystream/blob/0b8c3f8bdd6539a408bfcf6d9d4539a53cbb7649/storage-node/src/services/webApi/app.ts#L105). Without this change if any of the route handlers (e.g. handler for `/status`) threw an error then it would crash the application since it would not be caught anywhere.

It's the equivalent of following piece of code in distributor node
https://github.com/zeeshanakram3/joystream/blob/0b8c3f8bdd6539a408bfcf6d9d4539a53cbb7649/distributor-node/src/services/httpApi/HttpApiBase.ts#L27-L31